hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/container/balancer/TestContainerBalancer.java

```diff
@@ -235,16 +235,16 @@ public void testStartBalancerStop() throws Exception {
     startBalancer(balancerConfiguration);
     try {
       containerBalancer.startBalancer(balancerConfiguration);
+      Assertions.assertTrue(false);
```

Comment:
Lets also add a message in the assertion like "An exception should have been thrown"